src/vs/workbench/contrib/inlineChat/browser/inlineChatWidget.ts

```diff
@@ -464,6 +465,14 @@ export class InteractiveEditorWidget {
 		this._preferredExpansionState = expansionState;
 	}
 
+	set indentationWidth(indentationWidth: number) {
+		this._indentationWidth = indentationWidth;
+	}
+
+	get indentationWidth(): number {
+		return this._indentationWidth;
+	}
```

Comment:
I'd say this should belong inside `InteractiveEditorZoneWidget` because the widget itself (this class) shouldn't have to deal with the layouting constrains/trick of its container.

Reply:
But the indentation width